sdk/storage/storage-blob/README.md

```diff
@@ -66,7 +69,7 @@ There are differences between Node.js and browsers runtime. When getting start w
 
 ### NPM
 
-The preferred way to install the Azure Storage SDK for JavaScript is to use the npm package manager. Simply type the following into a terminal window:
+The preferred way to install the Azure Storage library for JavaScript is to use the npm package manager. Simply type the following into a terminal window:
```

Comment:
`Azure Storage library` ?
storage blob library ?

Reply:
want to keep the product name capitalized.

---
In reply to: [295981718](https://github.com/Azure/azure-sdk-for-js/pull/3744#discussion_r295981718) [](ancestors = 295981718)

Reply:
ah thought you were talking about casing. fixing.